src/docs/about.md

```diff
@@ -0,0 +1,186 @@
+runner), [Gulp](https://gulpjs.com/) (streams and pipes) or [Webpack](https://webpack.js.org/) (a module bundler),
+these all operate on different levels to Broccoli.
+
+Broccoli works at the file-system level, it provides a Javascript API to wrap other node tools like
+[Babel](https://babeljs.io/), [Rollup](https://rollupjs.org/) or [Node-Sass](https://github.com/sass/node-sass), that
+operate on a set of input files, and emit output files, that is all. Broccoli has no knowledge of the contents of 
+your files, only input and output directories.
+
+## Thinking in Broccoli
+
+Broccoli at its heart is very simple. It's really just a set of connected file transformations, performed by plugins,
+that manipulate files from one or multiple inputs, into a single output. Each plugin is responsible for processing files
+passed to it in input directories, and writing files to its output directory.
+
+Broccoli handles managing the file-system state, and simply takes input files and passes them as inputs to plugins, 
+and takes the output from each plugin and passes it as an input to the next one. The order in which these operations 
+happen is determined by the build file, a `Brocfile.js` that you're going to learn how to create.
+
+You can think of Broccoli much like a simple programming language, where the output of a function can be passed as the
+input(s) to another function.
+
+E.g.:
+
+```js
+module.exports = merge([
+    uglify(
+        babel('src')
+    ),
+    autoprefixer(
+        sass('src/styles', 'site.scss')
+    )
+]);
+```
+
+In the above, working from inner to outer, a `src` directory is passed to the `babel()` plugin (which will convert our
+new ES6 syntax into ES5 that runs in the browser), the output of that is passed to the `uglify()` plugin which will
+minify our JS into a smaller format. The output of `uglify()` will in turn be passed as 1 input to `merge()`.
+
+Additionally, a `src/styles` directory and the input file `site.scss` is passed to `sass()` which will convert your
+`.scss` files into `.css` files, its output (your css) is passed as an input to `autoprefixer()` which will add vendor 
+prefixes (like `-ms` or `-webkit`) to attributes, which is then in turn passed into the `merge()` plugin.
+
+Merge will copy the contents of each of its inputs into its output directory. Thus, it merges our uglified Javascript
+and out vendor prefixed css. This then becomes our final output and is what is written to our target (destination) 
+directory.
+
+This should all be fairly familiar to you if you've ever written Javascript (or any programming language for that
+matter) before, it's just inputs and output.
+
+## Thinking in Broccoli
+
+There are 3 main concepts to get your head around when using Broccoli:
+
+* [Plugins](#plugins)
+* [Nodes](#nodes)
+* [Trees](#trees)
+
+### Plugins
+
+Plugins are what a build pipeline developer will interact with most. Plugins are what do the actual work of transforming
```

Comment:
We spoke about this in todays call, I will review again once you had time to iterate on this section.